pkgs/servers/matrix-corporal/default.nix

```diff
@@ -0,0 +1,26 @@
+{lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "matrix-corporal";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "devture";
+    repo = pname;
+    rev = version;
+    sha256 = "0xmfdqvyl956s2y0qa7mcbkbcdzpicjxxxaz181gwdin8h9mdj0p";
+  };
+
+  buildFlagsArray = [
+    "-ldflags=-X main.GitCommit=${version} -X main.GitBranch=${version} -X main.GitState=nixpkgs -X main.GitSummary=${version} -X main.Version=${version}"
```

Comment:
```suggestion
    "-ldflags=-s -w -X main.GitCommit=${version} -X main.GitBranch=${version} -X main.GitState=nixpkgs -X main.GitSummary=${version} -X main.Version=${version}"
```

Reply:
Do we always strip debug information in nixpkgs?

I just copied the ones from https://nixos.org/manual/nixpkgs/unstable/#ssec-go-common-attributes which doesn't